woocommerce/woocommerce-ios
# Add multiple button support to TopBannerView

closes #2646 

# Why
For the new products feedback survey, we need to modify the products top banner. Prior this PR the `TopBannerView` class only supported one action button.

# How
- Adds a new `ActionButton` struct inside `TopBannerViewModel` that will represent an action button.
- Update `TopBannerView` to build its view hierarchy based on the new `actionButtons` property from `TopBannerViewModel`.
- Adds some tests.

# Screenshot
This is not in the app, but is only to show you how it would look.
<img width="395" alt="mutile-layout-button" src="https://user-images.githubusercontent.com/562080/90651589-cd46c280-e202-11ea-8090-5e1f45820f1c.png">

# Testing Steps
Make sure that the current banners(Products and Stats) keep looking as they used to.
* To show the stats banner you may need to replace line 45 of `StatsVersionCoordinator` with
```swift
let nextVersion: StatsVersion = .v3
```

Update release notes:
